### DISCUSSION TITLE
Is Dust 3D dead?

### DISCUSSION THREAD

**Discussion by nonprogrammodaunpo** [huxingyi/dust3d#134] [General]
I stumbled upon Dust 3D and found it the best for unleashing creativity!
But then I noticed every development stopped 3 years ago.
There were some answers to discussions until 2021, then nothing...

Since I'd love to have such an easy 3D tool for my modeling, I wonder if I better look elsewhere, or should keep waiting for Dust 3D to be maintained and improved...

thanks a lot for the answer, Jeremy!

**Comment by kidandcat** [comment]
The new features and experimentation is happening in this repo: https://github.com/huxingyi/tubetube
